### PR TITLE
Meeting notes from 09/21-09/25/2020

### PR DESCRIPTION
**Pull request by michdolan**
Notes for the latest TSC meeting and config working group. please suggest edits if I misheard or missed anything.

Signed-off-by: Michael Dolan <michdolan@gmail.com>

### CODE REVIEW

File: ASWF/meetings/tsc/2020-09-21.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    - Patrick: Already have a problem with one dependency not using 
+      CMAKE_BUILD_TYPE.
+    - Mark: On Windows doesn't matter at all. XCode generator has similar 
+      problem on macOS.
+    - Patrick: Document on Windows for now, and add something missing if we 
+      find better solution later. New VS 2019 also introducing problems by 
+      itself.
+    - Thomas: Should follow VFX platform, mandates VS version now. If you want 
+      to be compliant on Windows, use 2017.
+    - Michael: CI VMs use 2019, no option to use other version without a
```
[the remark this answers is not in the document]
Comment (michdolan) [reply ASWF/meetings/tsc/2020-09-21.md:78]: Awesome. I'll make note of this.

[the remark this answers is not in the document]
Comment (michdolan) [reply ASWF/meetings/tsc/2020-09-21.md:78]: Added

File: ASWF/meetings/tsc/2020-09-21.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    - Thomas: Should follow VFX platform, mandates VS version now. If you want 
+      to be compliant on Windows, use 2017.
+    - Michael: CI VMs use 2019, no option to use other version without a 
+      container. hopefully can find solution in CI working group.
+
+* TSC opinions on C++/Python API Docs Generation (Issue #1148):
+    - Sean: Just RTD was problematic. Everything else working well. All 
+      features listed in issue work well locally, but RTD can't do what we 
+      need. Other projects have made hacks with skeleton python bindings, but 
+      unless you automate it drifts.
```
[the remark this answers is not in the document]
Comment (michdolan) [reply ASWF/meetings/tsc/2020-09-21.md]: Fixed

File: ASWF/meetings/tsc/2020-09-21.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    - Doug: One thing discussing in documenting wg - we want to know who our 
+      customer is. Feedback is that biggest customer of web based API docs is 
+      python developers since C++ devs will often just go to the headers and 
+      dev env. If Python user is target, and since there are some significant 
+      differences, question came up as to whether we should target python 
+      specifically. IS C++ docs needed? Question to raise, not suggesting one 
+      or the other.
+    - Carl: As C++ dev, we generally just look at header since html docs awe
```
[the remark this answers is not in the document]
Comment (michdolan) [reply ASWF/meetings/tsc/2020-09-21.md]: Fixed

**Review by doug-walker — APPROVED** [review]

File: ASWF/meetings/tsc/2020-09-21.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    - Mark: RTD business account could solve it.
+    - Sean: Not sure if we can build all of OCIO. Would have to do dependencies.
+    - John: Might be a good Andy question on RTD. Most projects using free 
+      tier, but if it's something projects want, can investigate. Some services 
+      have non-profit discounts.
+    - Carl: OIIO is on RTD; should check with Larry.
+    - Thomas: Basic price $50/month. Advanced $100/month.
+    - Michael: Any objection to Sean's approach with new deployment method? 
+    - Doug: Any current example of Python API override? Transitioning from 
+      previous system, so unless you see the output, its hard ot make final
```
Comment (hodoulp) [inline ASWF/meetings/tsc/2020-09-21.md:139]: 'hard ot make' -> 'hard to make'

**Comment by hodoulp** [comment]
Approved the TSC meeting minutes.

File: ASWF/meetings/tsc/2020-09-21.md (hunk 4 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2020-09-21.md]: Fixed.

**Comment by hodoulp** [comment]
As usual this is a notice that we will merge this on Tuesday, if there are no more comments on the pull request. But I would rather prefer to have someone present at the Config meeting approving the pull request.

**Review by hodoulp — APPROVED** [review]

**Pull request merged**